app/assets/javascripts/general.js

```diff
@@ -42,6 +42,163 @@ $('#standard-popup .js-popup__close').click(function() {
   return false;
 });
 
+// tooltips
+$(document).ready(function(){
+
+   /*
+    * jQuery simple and accessible dialog tooltip window, using ARIA
```

Comment:
This looks like it should live in its own file, with just the library configuration in `general.js`?